CMakeLists.txt

```diff
@@ -751,15 +751,15 @@ set(PACKAGE_URL "https://github.com/opencollab/arpack-ng/")
 set(ARPACK_PC_LIBS_PRIVATE)
```

Comment:
ARPACK_PC_LIBS_PRIVATE looks like to be unused in CMakeLists.txt !?... Do you use it? If no, kill it

Reply:
I'm not sure I understand what you mean.
It is used in `SRC/arpack.pc.in`.

Reply:
OK, not used in CMakeLists.txt but used elsewhere!